### DISCUSSION TITLE
Issue when using UsePaveBasis in TID

### DISCUSSION THREAD

**Discussion by andlessa** [FeynCalc/feyncalc#217] [Q&A] [answered]
Hi,

I'm trying to compute a 4-point function at 1-loop (box diagram) and after simplifying the function I try to do the tensor decomposition in terms of the PaVe functions using UsePaVeBasis->True.
However some of the terms in the output still contain the loop  momentum and  the propagators containing the loop variable.
Is this the expected behavior? Am I missing something here? 
Here is the output:

![Screenshot from 2023-04-03 21-45-44](https://user-images.githubusercontent.com/5964046/229659325-1cc7374a-67c0-449f-9922-8a4b87507b7e.png)

The full notebook is here:

[bugExample.pdf](https://github.com/FeynCalc/feyncalc/files/11143695/bugExample.pdf)

**Comment by vsht** [comment]
Hi,

just add the option `ToPaVe->True` to TID.

**Reply by andlessa** [reply]
But in this case I will get the decomposition in terms of scalar integrals, right? What I would like is the decomposition in terms of the tensor integrals, so the resulting expression is not so messy.

**Comment by vsht** [comment, marked as answer]
No, having both options set to True will give you exactly what you want. Just try it ;)

Am 6. April 2023 17:31:01 UTC schrieb Andre Lessa ***@***.***>:
